katalog/node-exporter/clusterRole.yaml

```diff
@@ -1,17 +1,17 @@
-# Copyright (c) 2020 SIGHUP s.r.l All rights reserved.
+# Copyright (c) 2022 SIGHUP s.r.l All rights reserved.
 # Use of this source code is governed by a BSD-style
 # license that can be found in the LICENSE file.
 
----
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: node-exporter
----
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
+  labels:
+    app.kubernetes.io/component: exporter
+    app.kubernetes.io/name: node-exporter
+    app.kubernetes.io/part-of: kube-prometheus
+    app.kubernetes.io/version: 1.3.1
   name: node-exporter
+  namespace: monitoring
```

Comment:
Since we are importing the manifests from the upstream project, the namespace is already specified. I didn't remove it to keep the number of changes from upstream at minimum.